Resources/Prototypes/Recipes/Reactions/drinks.yml

```diff
@@ -777,6 +777,16 @@
   products:
     SyndicateBomb: 2
 
+- type: reaction
```

Comment:
It would be interesting, though certainly not a requirement for this PR, if you required heating to make tea.